eland/operations.py

```diff
@@ -1273,6 +1273,8 @@ def to_json(  # type: ignore
             for i, df in enumerate(
                 self.search_yield_pandas_dataframes(query_compiler=query_compiler)
             ):
+                if pd.__version__.split(".")[0] == "2":
+                    kwargs["index"] = False
```

Comment:
What's wrong with the `None` default? Also, in 1.5, the default is `True`: why choose `False` here?

Reply:
I'm not sure anymore why I added it, I'll remove it locally and see what might go wrong without these two lines.

Reply:
I slightly changed this, so `None` is the default now (except if Pandas < 2 is installed).

There was nothing wrong with the default :-)